bench/bench.ml

```diff
@@ -491,6 +528,11 @@ let json_flag =
   let env = env_var "JSON" in
   Arg.(value & flag & info [ "j"; "json" ] ~env ~doc)
 
+let frequency =
+  let doc = "The frequency of timing the replace operations." in
```

Comment:
'frequency' to me implies dimensions of `time^{-1}` (i.e. bigger numbers correspond to more data points). Perhaps something like 'sampling interval' is a better term.